### PR TITLE
SAK-45731 Improve a11y of timer in samigo

### PR DESCRIPTION
**Pull request by stetsche**
Screen reader text:

- Remaining time will be announced about every minute (Will still be read on ctrl+alt+t keybinding)
- Text is now localized.
- Time announced has a more compact format

Progress bar colors:

- Reduced to 3 from 100
  - 100%-51% green
  - 50%-25% orange
  - 24%-0% red

Remaining time countdown:

- Seperated from progress bar
- "Remaing Time" text removed, just time is displayed
- Sufficient contrast
- light theme [8.41:1](https://webaim.org/resources/contrastchecker/?fcolor=333435&bcolor=D3D4D6) 
- dark theme [4.55:1](https://webaim.org/resources/contrastchecker/?fcolor=D3D4D6&bcolor=555C65) 

Alert at 10% of time: Removed

Hide/Show time remaining is now a button

### CODE REVIEW

**Comment by stetsche** [comment]
@mpellicer Can you take a look please?

File: samigo/samigo-app/src/webapp/jsf/delivery/getTimerStrings.jsp [hunk 1]
```diff
@@ -2,5 +2,5 @@
...
+<h:outputText value="{&quot;savedMessage&quot;:&quot;#{deliveryMessages.savedMessage}&quot;,&quot;subMessage&quot;:&quot;#{deliveryMessages.subMessage}&quot;,&quot;hideMessage&quot;:&quot;#{deliveryMessages.hideMessage}&quot;,&quot;showMessage&quot;:&quot;#{deliveryMessages.showMessage}&quot;,&quot;srRemaining&quot;:&quot;#{deliveryMessages.srRemaining}&quot;,&quot;srTimerInfo&quot;:&quot;#{deliveryMessages.srTimerInfo}&quot;,&quot;minReqScale&quot;:&quot;#{delivery.minReqScale}&quot;}" escape="false"/>
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/jsf/delivery/getTimerStrings.jsp:5]: Ouch, Yuck!

File: samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js [hunk 2]
```diff
@@ -41,23 +33,44 @@
...
+              </div>
+              <button id='showHide'>
+                  <span class='showHideSym'>▲</span>
+                  <span id='showHideText'>${ hideMessage }"</span>
+                  <span class='showHideSym'>▲</span>
+              </button>
+          </div>
+          <div id='dialog-timeout'>
+              <div id='indicator'>
+                  <img class='ajaxImg' src='/samigo-app/images/ajaxReq.gif'>
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js:69]: Good catch!

**Review by mpellicer — CHANGES_REQUESTED** [review]
Awesome work and nice documentation @stetsche.

Fix the issues reported by AccessLint and also try to fix the Codacy complaints if possible.

Do you think recording a video or a GIF file with the visual changes makes sense? To complement the Jira documentation.

Thanks

**Review by mpellicer — APPROVED** [review]

**Pull request merged**